components/style/properties/longhand/inherited_text.mako.rs

```diff
@@ -60,16 +60,18 @@ ${helpers.single_keyword("word-break",
                          spec="https://drafts.csswg.org/css-text/#propdef-word-break")}
 
 // TODO(pcwalton): Support `text-justify: distribute`.
-<%helpers:single_keyword_computed name="text-justify"
-                                  values="auto none inter-word"
-                                  extra_gecko_values="inter-character"
-                                  extra_specified="${'distribute' if product == 'gecko' else ''}"
-                                  gecko_enum_prefix="StyleTextJustify"
-                                  animation_value_type="discrete"
-                                  gecko_pref="layout.css.text-justify.enabled"
-                                  flags="APPLIES_TO_PLACEHOLDER",
-                                  spec="https://drafts.csswg.org/css-text/#propdef-text-justify">
-
+<%helpers:single_keyword
```

Comment:
Can we change this to the:

```
${helpers.single_keyword(...)}
```

syntax while you're here? I think we use it more often.

(followup / E-easy is fine too if you prefer).

Reply:
Can't do that, given I need to implement `ToComputedValue` by hand for the Gecko product.